.github/workflows/build-fuzz-reusable.yml

```diff
@@ -0,0 +1,74 @@
+---
+name: Build - Fuzztests on L0 HW - Reusable
+
+on:
+  workflow_call:
+    inputs:
+      test_name:
```

Comment:
`test_label` to be precise

Reply:
Yes, that fits better, thanks.

Reply:
heh, pls fix also usage of this var, now 😉

Reply:
Done